src_web/kamaho-shokusu/src/Policy/AiAssistantPolicy.php

```
@@ -0,0 +1,17 @@
+declare(strict_types=1);
+
+namespace App\Policy;
+
+use Authorization\IdentityInterface;
+
+class AiAssistantPolicy
+{
+    /**
+     * AIへの質問を許可するかどうか。ログインしていれば許可する。
+     */
+    public function canAsk(IdentityInterface $user, $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)